ui/raidboss/data/04-sb/dungeon/doma_castle.txt

```diff
@@ -95,48 +95,48 @@ hideall "--sync--"
 2009.4 "Chainsaw" duration 5.5 Ability { id: "20A8", source: "Hypertuned Grynewaht" }
 2022.7 "--center--"
 2022.7 "Delay-Action Charge" Ability { id: "20AD", source: "Hypertuned Grynewaht" } window 10,10 # boss walks center before using
-2023.1 "Gunsaw" duration 5.0  Ability { id: "20AA", source: "Hypertuned Grynewaht" }
+2023.1 "Gunsaw" Ability { id: "20AA", source: "Hypertuned Grynewaht" } duration 5.0
```

Comment:
> Here and elsewhere, it might be worth a cleanup pass to convert any unit floats to ints in a future PR.

Yeah, that's a good thought.  I'll do a follow-up PR after this lands -- if for no other reason than to avoid touching even more files and forcing one of you to review even more changes.

Just to clarify though -- I'd be inclined to flag lint for any ".0" floats for `window`/`duration`/`jump`/`forcejump`, maybe even `label`.  But for actual syncs, I'd sorta lean toward leaving them alone?  I think `make_timeline` always auto-generates sync entries as floats, and for consistency/readability, it might look weird to have a long list of floats with a few random ints interspersed.  But lmk if you disagree.

Reply:
Oh, yes, sorry, I should have been clearer. Your understanding is correct, the sync timestamps should always be floats regardless.